components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -488,11 +489,11 @@ var img = "";
 							      
 							      	       <tr>
 							      	      	   <td>
-							      	      		<input name="step_<%=step.getStepOrder()%>_fed_auth" id="" type="hidden" value="<%=idp.getIdentityProviderName()%>" />
-							      	      			<%=idp.getIdentityProviderName() + (idpEnableStatus.get(idp.getIdentityProviderName()) != null && idpEnableStatus.get(idp.getIdentityProviderName()) ? "" : disbleText) %>
+							      	      		<input name="step_<%=step.getStepOrder()%>_fed_auth" id="" type="hidden" value="<%=Encode.forHtmlAttribute(idp.getIdentityProviderName())%>" />
+							      	      			<%=Encode.forHtmlAttribute(idp.getIdentityProviderName()) + idpEnableStatus.get(Encode.forHtmlAttribute(idp.getIdentityProviderName())) != null && idpEnableStatus.get(Encode.forHtmlAttribute(idp.getIdentityProviderName())) ? "" : disbleText %>
 							      	      		</td>
 							      	      		<td> 
-							      	      			<select   name="step_<%=step.getStepOrder()%>_idp_<%=idp.getIdentityProviderName()%>_fed_authenticator" style="float: left; min-width: 150px;font-size:13px;"><%=stepIdpAuthenticators.get(step.getStepOrder() +"_"+ idp.getIdentityProviderName())%></select>
+							      	      			<select   name="step_<%=step.getStepOrder()%>_idp_<%=idp.getIdentityProviderName()%>_fed_authenticator" style="float: left; min-width: 150px;font-size:13px;"><%=Encode.forHtmlAttribute(stepIdpAuthenticators.get(step.getStepOrder()) +"_"+ Encode.forHtmlAttribute(idp.getIdentityProviderName()))%></select>
```

Comment:
Should be as below
<select   name="step_<%=step.getStepOrder()%>_idp_<%=Encode.forHtmlAttribute(idp.getIdentityProviderName())%>_fed_authenticator" style="float: left; min-width: 150px;font-size:13px;"><%=Encode.forHtmlContent(stepIdpAuthenticators.get(step.getStepOrder()) +"_"+ Encode.forHtmlContent(idp.getIdentityProviderName()))%></select>